File: PythonAbril30dias/dia8.py
name = input("Introduce tu nombre: ").lower()
if len(name) > 7:
    print(f"¡Vaya, {name.capitalize()}, ¡qué nombre tan largo y sofisticado!.")
elif name.startswith("a"):
    print(f"¡Hola!, Asombroso/a {name.capitalize()}")
else:
    print(f"Saludos, {name}")

File: JavaAbril30dias/src/Dia8.java
import java.util.Scanner;

public class Dia8 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Introduce un nombre:");
        String name = input.nextLine().toLowerCase();
        if (name.length() > 7){
            System.out.println("¡Vaya, " + name.substring(0, 1).toUpperCase() + name.substring(1) + ", ¡qué nombre tan largo y sofisticado!.");
        }
        else if (name.charAt(0) == 'a') {
            System.out.println("¡Hola, Asombroso/a " + name.substring(0, 1).toUpperCase() + name.substring(1) + "!.");
        }
        else {
            System.out.println("Saludos, " + name.substring(0, 1).toUpperCase() + name.substring(1));
        }
    }
}
